main.py

```diff
@@ -1,13 +1,105 @@
-#!/usr/bin/env python
+import json
+import pandas as pd
+from nltk import word_tokenize
+from nltk.corpus import stopwords
+from stop_words import get_stop_words
+from nltk.wsd import lesk
+import re
 
-# Your classes or methods go here:
 
+class Writer:
 
-class Tool:
-    def my_method(self):
-        print("my_method called!")
+    """Writer data to json file. Read txt file"""
+
+    @staticmethod
+    def write_to_json(data_text):
+        with open('task.json', 'w') as json_w:
```

Comment:
Remember, you shouldn't hardcode such things a file name - this should be configurable. You may provide it as a default value though.